compiler/src/dmd/libmach.d

```diff
@@ -485,7 +485,14 @@ struct MachObjModule
     uint length; // in bytes
     uint offset; // offset from start of library
     const(char)[] name; // module name (file name) with terminating 0
+version (OpenBSD)
+{
+    long file_time; // file time
+}
+else
+{
     c_long file_time; // file time
```

Comment:
```suggestion
    time_t file_time; // file time
```
Can't you import this or infer it from druntime?

Reply:
Will that work for all platforms? I have no idea.

Reply:
The field is currently passed by pointer to `time`:

```
.time(&om.file_time);
```

And the signature of `time` is:

```D
time_t  time(scope time_t* timer);
```

If that would fail, then so would this PR.